### DISCUSSION TITLE
How would you style components with a style prop that has multiple props to style different parts of the component

### DISCUSSION THREAD

**Discussion by andredewaard** [nativewind/nativewind#940] [Q&A]
[this](https://github.com/lawnstarter/react-native-picker-select?tab=readme-ov-file) component has one style prop that needs an object with multiple style objects.
you would write it like:
```
            style={{
              inputAndroid: {
                backgroundColor: 'transparent',
              },
              iconContainer: {
                top: 5,
                right: 15,
              },
            }}
``` 
How can we handle this with nativewind remapProps?

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
